client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
@stephanethomas : I like this. Seems like it solves both concerns by suffixing the action, which allows each area of concern to be easily scannable while still preserving alphabetization. I also agree that standardizing on `REQUEST` instead of `FETCH` may be a wise choice.